internal/brokers/broker.go

```diff
@@ -67,12 +67,22 @@ type fieldValidator struct {
 	required        bool
 }
 
+// BrokerID computes the broker ID from its name.
+func BrokerID(brokerName string) string {
```

Comment:
I don't like the idea of making this public just for the sake of one test case. Can we make it private and then access the broker ID through the broker object instead?

Reply:
Mh, I wasn't in love either, but it's the only way I know to be able to predict this value...

I've pushed another solution though, not the cleanest either, but making possible to select a broker from its name too from tests.